input/pagecontent/index.md

```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Medicine Information](medicine-information.html): This page provides guidance on constructing medications related resources.
   - [AUCDI](aucdi.html): This page maps the AU Core resources and elements to AUCDI data classes and data elements.
   - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Core and other implementation guides.
-  - [Comparison with other national and international specifications](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [AU Variance Statement](variance.html): This page documents the variance from AU Base.
   - [Comparison with other national and international IGs](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [Future of AU Core](future.html): This page outlines the approach to developing AU Core and yearly update cycle.
+- [Security and Privacy](security.html): This page outlines the security and privacy obligations to consider when implementing AU Core.
```

Comment:
obligations is a loaded word, do not use unless you mean FHIR Obligations. These are requirements. Use the below text instead.

"This page documents the AU Core general security and privacy requirements and recommendations."

Reply:
Fixed, thanks.